static/js/stripe-app.js

```diff
@@ -27,19 +22,23 @@ $(window).load(function () {
                         error: function(err) {
                             console.error("Error updating user.");
                             console.error(err);
-                        },
-                        complete: function(){
-                            document.location.reload();
                         }
                     });
                 }
+                $('#stripe-form').prepend(message_array.join(""))
+                $('#stripe-submit').text('Encrypt Account Information');
             }
         })(this)
     }
 
     let handleResponse = new stripeResponse($('form input[name="csrfmiddlewaretoken"]').val())
     let $form = $('#stripe-form')
     let account_type = $form.attr('stripe-account-type')
+    let message_array = ['<div class="callout warning expanded" data-closable>',
+        '<button class="close-button" data-close>&times;</button>',
+        '<p class="alert alert-error">',
+        'MESSSAGE GOES IN POS 3',
```

Comment:
lulz. possibly the worst JS templating engine of all time? Chrome, Opera, and Firefox all support [JS Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Browser_compatibility), so something like this should work:
```
let message = response.error.message
let messageTemplate = `<div class="callout warning expanded" data-closable><button class="close-button" data-close>&times;</button><p class="alert alert-error">${message}</p></div>`
```